nix/haskell.nix

```
@@ -5,7 +5,7 @@
 , buildPackages
 , config ? {}
 # GHC attribute name
-, compiler ? config.haskellNix.compiler or "ghc883"
+, compiler ? config.haskellNix.compiler or "ghc8101"
```

this won't change it to ghc 8.10 I think because `config.haskellNix.compiler` is not null.
```suggestion
, compiler ? "ghc8101"
```